src/components/NavBar/index.tsx

```diff
@@ -76,19 +69,22 @@ const NavBar = function () {
               <div className={navBarStyles.hpLogoText}>HyperPlay</div>
             </div>
           </a>
-          <button
-            className={navBarStyles.burgerMenu}
-            onClick={() => setShowNavBarDropDown(!showNavBarDropDown)}
-          >
-            <img
-              src={burgerMenuIcon}
-              //fill={true}
-              style={{ objectFit: 'contain' }}
-              alt="Menu Button"
-            />
-          </button>
+          <div className={navBarStyles.dropdownContainer}>
+            {UserAvatar}
+            <button
+              className={navBarStyles.burgerMenu}
+              onClick={() => setShowNavBarDropDown(!showNavBarDropDown)}
+            >
+              <img
+                src={burgerMenuIcon}
+                //fill={true}
+                style={{ objectFit: 'contain' }}
```

Comment:
obligatory let's move this to the sass module comment 😄 
also let's remove the //fill comment line

you could add an any child img selector to burgerMenu to avoid creating a new class too